core/arch/arm/plat-rpi3/platform_flags.mk

```diff
@@ -0,0 +1,20 @@
+# 32-bit flags
```

Comment:
Ok, seems like we're all on the same page here. I'll remove this one and I'll add a separate PR for the `ta/ta.mk` later on.